tests/integration-tests/clusters_factory.py

```diff
@@ -156,15 +155,15 @@ def cfn_name(self):
     @property
     def region(self):
```

Comment:
Shall we pass region as an arg to the Cluster __init__?

Reply:
Done